### PR TITLE
fix(charts): show Brak danych for zero-data time-series

### LINKED ISSUE #801: fix(charts): zero-data time-series show 0k/0k axis instead of empty state
**Where:** Metryki › "Wzrost według typu konta/inwestycji" — e.g. "IKZE w czasie", "Obligacje w czasie".

**Problem:** with no holdings the chart draws a flat line at zero and a degenerate y-axis (`0k / 0k / 0k / 0k`). Looks broken.

**Fix:** detect all-zero/empty series and render a "Brak danych" empty state instead of the zero-line + axis.

Ref: `docs/ux-ui-review-2026-06.md`. Part of #798.

> Changelog: skip

### PR DESCRIPTION
**Pull request by Automaat**
## Motivation

Time-series charts for accounts/categories with no holdings (e.g. IKZE, Obligacje before any entries) rendered as a degenerate flat-line chart at 0k/0k. This was misleading — it looked like data existed but was zero, rather than communicating that no data is available yet.

Closes https://github.com/Automaat/finance-buddy/issues/801

> Changelog: fix(charts): show Brak danych for zero-data time-series

## Implementation information

- `buildTrendChartOption` in `src/lib/utils/charts/metryki.ts` now detects when all data points are zero (or there are no data points) and returns an ECharts option with hidden axes and a centered 'Brak danych' graphic instead of the degenerate chart.
- Tests in `metryki.test.ts` updated: the 'handles an empty input' case now asserts the empty-state option shape (no series, graphic defined); the 'defaults missing values to zero' test was adjusted to use a non-zero value so it tests field-level defaulting without triggering the empty-state path; new tests cover the all-zero series case.

### CODE REVIEW

File: src/lib/utils/charts/metryki.ts [hunk 1]
```diff
@@ -233,6 +233,27 @@ function buildTrendChartOption(
...
+			graphic: [
+				{
+					type: 'text',
+					left: 'center',
+					top: 'middle',
+					style: { text: 'Brak danych', fontSize: 16, fill: chartInkMuted }
+				}
+			]
+		};
+	}
```
[the remark this answers is not in the document]
Comment (Automaat) [reply src/lib/utils/charts/metryki.ts:256]: **Applied** — replaced `xAxis: { show: false }` / `yAxis: { show: false }` with `xAxis: []` / `yAxis: []` and added `legend: { data: [] }`. Empty arrays replace the component list in ECharts merge mode, so axes and legend don't persist when data returns (cbbef2a).

File: src/lib/utils/charts/metryki.test.ts [hunk 2]
```diff
@@ -94,17 +94,34 @@ describe('buildInvestmentTrendChartOption', () => {
...
 
-	it('handles an empty input', () => {
+	it('returns empty state for empty series', () => {
 		const option = buildInvestmentTrendChartOption([]);
-		const series = option.series as Array<{ data: number[] }>;
-		expect(series[0].data).toEqual([]);
+		expect(option.series).toEqual([]);
+		const graphic = option.graphic as Array<{ style: { text: string } }>;
+		expect(graphic[0].style.text).toBe('Brak danych');
+	});
```
[the remark this answers is not in the document]
Comment (Automaat) [reply src/lib/utils/charts/metryki.test.ts:112]: **Applied** — all 5 empty-state tests now assert `xAxis: []`, `yAxis: []`, and `legend.data: []` in addition to the existing `series` and `graphic` checks (cbbef2a).
